Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -326,15 +361,7 @@ DECLARE_SOA_INDEX_COLUMN_FULL(Bachelor, bachelor, int, Tracks, "fTracksID");
 DECLARE_SOA_TABLE(Cascades, "AOD", "CASCADE", cascade::V0Id, cascade::BachelorId);
 using Casecade = Cascades::iterator;
 
-namespace trigger
-{
-DECLARE_SOA_COLUMN(GlobalBC, globalBC, uint64_t);
-DECLARE_SOA_COLUMN(TriggerMask, triggerMask, uint64_t);
-} // namespace trigger
-
-DECLARE_SOA_TABLE(Triggers, "AOD", "TRIGGER", trigger::TriggerMask, trigger::GlobalBC);
-using Trigger = Triggers::iterator;
-
+// FIXME can the timeframe table be removed?
```

Comment:
Please comment

Reply:
I don't like these names: aDTimeA, aDTimeC etc. Alternative proposal:
timeV0A, timeADA, timeADC, timeT0A, timeT0C
amplitudeV0A, amplitudeADA,...
triggerSignalV0A, trigerSignalADA,

Reply:
By the way, we should keep in mind that they renamed AD into FDD (Forward Diffractive Detector). May be we should better write: timeFDA, timeFDC etc...

Reply:
@ekryshen I agree. Even timeFDDA, right?